File: examples/installation/ios/objc/CocoaPodsExample/CocoaPodsExampleTests/CocoaPodsExampleTests.m
//
//  CocoaPodsExampleTests.m
//  CocoaPodsExampleTests
//
//  Created by JP Simard on 5/6/15.
//  Copyright (c) 2015 Realm. All rights reserved.
//

#import <UIKit/UIKit.h>
#import <XCTest/XCTest.h>
#import "AppDelegate.h"

@interface CocoaPodsExampleTests : XCTestCase
@end

@implementation CocoaPodsExampleTests

- (void)testExample {
    XCTAssertTrue([MyModel isSubclassOfClass:[RLMObject class]]);
}

@end

File: examples/installation/ios/swift-2.0/CocoaPodsExample/CocoaPodsExampleTests/CocoaPodsExampleTests.swift
//
//  CocoaPodsExampleTests.swift
//  CocoaPodsExampleTests
//
//  Created by JP Simard on 5/6/15.
//  Copyright (c) 2015 Realm. All rights reserved.
//

import UIKit
import XCTest
import CocoaPodsExample
import RealmSwift

class CocoaPodsExampleTests: XCTestCase {
    func testExample() {
        // This is an example of a functional test case.
        XCTAssertNotNil(MyModel() as AnyObject is Object)
    }
}
